pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -324,3 +328,15 @@ func getJustification(r *spdx.Relationship) string {
 	}
 	return s
 }
+
+func isEmptyChecksum(v string) bool {
+	return map[string]bool{
+		// all 0 hash
+		"0000000000000000000000000000000000000000":                         true,
+		"0000000000000000000000000000000000000000000000000000000000000000": true,
+		// sha1 empty file
+		"da39a3ee5e6b4b0d3255bfef95601890afd80709": true,
+		// sha256 empty file
+		"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855": true,
```

Comment:
qq: do we need to handle sha512 also?

Reply:
yea and technically the whole list too :). so i just picked the common ones, but yea ill add sha512 for this one and then maybe the rest we can leave till when it comes up? or if something wants to help as a good first issue.

 [SHA1](https://tools.ietf.org/html/rfc3174), [SHA224](https://en.wikipedia.org/wiki/SHA-2), [SHA256](https://tools.ietf.org/html/rfc6234), [SHA384](https://en.wikipedia.org/wiki/SHA-2), [SHA512](https://en.wikipedia.org/wiki/SHA-2), [SHA3-256](https://en.wikipedia.org/wiki/SHA-3), [SHA3-384](https://en.wikipedia.org/wiki/SHA-3), [SHA3-512](https://en.wikipedia.org/wiki/SHA-3), [BLAKE2b-256](https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE2), [BLAKE2b-384](https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE2), [BLAKE2b-512](https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE2), [BLAKE3](https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE3), [MD2](https://tools.ietf.org/html/rfc1319), [MD4](https://tools.ietf.org/html/rfc1320), [MD5](https://tools.ietf.org/html/rfc1321), [MD6](https://groups.csail.mit.edu/cis/md6/), [ADLER32](https://en.wikipedia.org/wiki/Adler-32)

Reply:
added that and created an issue for tracking other algs (https://github.com/guacsec/guac/issues/1229)